### ISSUE TITLE
"Failed to save database"

### ISSUE THREAD

**Issue by tteguayco** [mifi/instauto#126]
Any idea why this may be happening?

**Comment by mifi** [comment]
maybe the path is wrong? what's your code like?

**Comment by tteguayco** [comment]
> maybe the path is wrong? what's your code like?

What path do you mean, @mifi? I'm running the example script that can be found in this repo, with a few tweaks on the options parameters.

**Comment by mifi** [comment]
Which folder are you running it from? Are you allowed to write files in that folder?

i meanthese:
https://github.com/mifi/instauto/blob/96b5fa837f3375aa0df1e1397eef0c9d97a72c56/example.js#L79

**Comment by tteguayco** [comment]
Yeah, indeed I was specifying a non-existing folder in this path, thus getting the error.

Maybe it would be better to make that log error message more verbose?

Thanks for the help, @mifi. Instauto is an awesome project!

**Comment by mifi** [comment]
Yes it would indeed! Pr welcome
